gninasrc/lib/model.h

```diff
@@ -426,12 +426,8 @@ struct model {
 
 	model() : m_num_movable_atoms(0), hydrogens_stripped(false), print_during_minimization(false), 
     eval_deriv_counter(0) {};
-	~model() { deallocate_gpu(); };

```

Comment:
why remove this?  deallocate_gpu should be safe to call even if it has already been deallocated explicitly.  Leaving it in the destructor is safe programming practice